#path_node to camera_node
float64 x #position.x
float64 y #position.y 
int32 state # mode (camera,path)
